### PR TITLE
Make tests pass on 0.6

### PR DESCRIPTION
**Pull request by daschw**
This works on my local machine and the new PyPlot testimages from https://github.com/JuliaPlots/PlotReferenceImages.jl/pull/5. I have no idea if it will work on travis, but it's worth a shot.

### CODE REVIEW

File: test/imgcomp.jl [hunk 1]
```diff
@@ -99,7 +99,7 @@ function image_comparison_facts(pkg::Symbol;
   for i in 1:length(Plots._examples)
     i in skip && continue
     if only == nothing || i in only
-      @fact image_comparison_tests(pkg, i, debug=debug, sigma=sigma, eps=eps) |> success --> true
+      @fact @eval(image_comparison_tests(Symbol(String(Symbol($pkg))[7:end]), $i, debug=$debug, sigma=$sigma, eps=$eps)) |> success --> true
```
Comment (daschw) [inline test/imgcomp.jl]: I don't really like `Symbol(String(Symbol($pkg))[7:end])` here. Maybe someone has a more elegant solution. `$pkg` returns e.g. for GR the backend `Plots.gr`.

**Comment by mkborregaard** [comment]
This looks strange, it looks as if it contains two of my commits: bad7772b1577f76c41385649414b3d53d5b073e4 and f462597dbd0dcc6da01801a1b87171dc005d9740 - but maybe we just had the same thought? :-)

**Comment by daschw** [comment]
no, i had to add the legendtitle because #892 was not merged and transpose errored on 0.6 so I had to do reshape.

**Comment by mkborregaard** [comment]
The Travis build fails, unfortunately. Any idea why?
What about if I merge my current 0.6 progress https://github.com/JuliaPlots/Plots.jl/pull/906 , might as well work directly on master with this?

**Comment by daschw** [comment]
Yes, thanks! I forgot to change the version in travis.yml

**Comment by mkborregaard** [comment]
> because #892 was not merged 

Yes, it was superseded by https://github.com/JuliaPlots/Plots.jl/pull/894 - I'll go ahead and rebase and merge that now so we don't fall over each other's feet :-)

**Comment by daschw** [comment]
Should I add another commit. Or do you want to merge and I rebase?

**Comment by mkborregaard** [comment]
already merged :-) You OK with me merging #906 and then you rebase on top of that?

**Comment by daschw** [comment]
sure

**Comment by mkborregaard** [comment]
great - so this PR is to adress the 'world age' problem, that ironically seem to only be affecting the tests, not the actual plots?

**Comment by daschw** [comment]
exactly

**Comment by daschw** [comment]
I think I will open a new clean PR, once you have merged #906.

**Comment by mkborregaard** [comment]
It's merged :-)

**Comment by daschw** [comment]
Closing in favour of #908

**Pull request closed without merging**